### PR TITLE
Specify proxy for UI plugin

### PR DESCRIPTION
**Pull request by rexagod**
Specify proxy for UI plugin that will be used to query prometheus over
TLS. Also bumped some dependencies to make
`openshift/api/console`.ConsolePluginProxy available.

Signed-off-by: Pranshu Srivastava <rexagod@gmail.com>

### CODE REVIEW

File: console/console.go [hunk 1]
```diff
@@ -82,6 +90,18 @@ func GetConsolePluginCR(consolePort int, basePath string, serviceNamespace strin
 				Port:      int32(consolePort),
 				BasePath:  basePath,
 			},
+			Proxy: []consolev1alpha1.ConsolePluginProxy{
+				{
+					Type:      consolev1alpha1.ProxyTypeService,
+					Alias:     tlsService + "-proxy",
```
Comment (SanjalKatiyar) [inline console/console.go:96]: we decided alias to be `mcg-managed-prom` ??
I am okay with `mcg-prometheus-proxy` as well, but it is something used in UI so need to be sure we are on same page.

Comment (rexagod) [reply console/console.go:95]: If it's okay can we pin it to `prometheus-proxy`? If you've already made changes using the above name on your end, then LMK and I'll revert this back to `mcg-managed-prom`.

Comment (SanjalKatiyar) [reply console/console.go:95]: okay... keeping it as `prometheus-proxy`... making same change in UI as well...

File: console/console.go [hunk 2]
```diff
@@ -31,6 +33,11 @@ const (
 	compatibilityBasePath = "/compatibility/"
 )
 
+var (
+	tlsService = "prometheus"
+	namespace  = "redhat-data-federation"
```
Comment (naveenpaul1) [inline console/console.go:38]: Method already passing namespace , do we need to hardcode this

**Review by naveenpaul1 — CHANGES_REQUESTED** [review]

**Comment by naveenpaul1** [comment]
LGTM

**Pull request merged**